pull-request/src/person.js

```diff
@@ -3,13 +3,14 @@ function Person(first, last) {
     this.lastName = last;
     this.age = null;
     this.eyeColor = null;
-    this.nationality = null;
+    this.nationality = undefined;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use null instead. ([javascript:S2138](http://localhost:9760/coding_rules#rule_key=javascript%3AS2138))

[See it in SonarQube](http://localhost:9760/project/issues?id=training%3Apull-request&pullRequest=52&issues=AWl8QWTu1Va9TISLgQ_2&open=AWl8QWTu1Va9TISLgQ_2)

[ISSUE_KEY]: AWl8QWTu1Va9TISLgQ_2